apache/superset
# fix: dropdown placement for cascading filters popover

### SUMMARY
This pr fixes an issue where the dropdown select will fill the container of the popover causing ui confusion for the user. This fix enables the drop down to drop in respect to the select

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before 
https://github.com/apache/superset/issues/16521#issue-983531427

after

https://user-images.githubusercontent.com/17326228/136634817-fa0f5d04-fe4c-42d2-9b67-4dddb2fd3178.mov



### TESTING INSTRUCTIONS
Go to dashboard and create cascading filter and ensure the the dropdown does not fill the popover container.

### ADDITIONAL INFORMATION


- [x ] Has associated issue: Fixes https://github.com/apache/superset/issues/16521
